xml/vt_introduction.xml

```diff
@@ -195,12 +195,9 @@
    </listitem>
    <listitem>
     <para>
-     Paravirtual mode does not require the host computer to support
-     <xref linkend="gloss.vt.hwassisted"/> virtualization technology, but
-     does require the guest operating system to be modified for the
-     virtualization environment. Typically, operating systems running in
-     paravirtual mode enjoy better performance than those requiring full
-     virtualization mode.
+     Paravirtual mode usually modifies guest operating system to suit better
+     in virtualization environment. Typically, operating systems running in 
+     paravirtual mode enjoy performance more than other stuff.
```

Comment:
Thanks for comment. It looks like it took hardware virtualization as Full virtualization here, however there is software emulation way in Full virtualization which uses Binary translation. I am trying to change it with minimum modification, but it looks more confuse now.